src/main/java/frc/robot/ControlInputs.java

```diff
@@ -16,11 +20,11 @@ public class ControlInputs {
     public double driveStickZrotation = 0.0;
 
     //Reading the controls
-    public final void readControls() {
+    public final void readControls(ComponentsControl componentsControl) {
         //Drivestick
-        driveStickX = driveStick.getLeftX() * Math.abs(driveStick.getLeftX());
-        driveStickY = driveStick.getLeftY() * Math.abs(driveStick.getLeftY());
-        driveStickZrotation = driveStick.getRightX() * Math.abs(driveStick.getRightX());
+        driveStickX = ( driveStick.getLeftX() * Math.abs(driveStick.getLeftX()) ) * driveStickMaxMovement;
+        driveStickY = ( driveStick.getLeftY() * Math.abs(driveStick.getLeftY()) ) * driveStickMaxMovement;
+        driveStickZrotation = ( driveStick.getRightX() * Math.abs(driveStick.getRightX()) ) * driveStickMaxRotation;
```

Comment:
I'm going to reiterate that the 'squaring' logic should be a small static helper function ;)

e.g.:
```java
driveStickX = squareInput(driveStick.getLeftX()) + driveStickMaxMovement;

[...]

/**
 * Square the input, preserving its sign
 */
public static double squareInput(double input) {
    return input * Math.abs(input);
}
```